### DISCUSSION TITLE
Bug : Language detection

### DISCUSSION THREAD

**Discussion by AshXD007** [community/community#55769] [Repositories]
### Select Topic Area

Bug

### Body

I have a private repo . It is a vuejs project , but github is detecting it as 98% HTML 1.6% Vuejs . What to do?

**Comment by airtower-luna** [comment]
Check the [Linguist troubleshooting documentation](https://github.com/github-linguist/linguist/blob/master/docs/troubleshooting.md). Linguist is what GitHub uses to detect the language(s) of your repository.
